sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -448,6 +449,42 @@ def is_buffered_correctly(actual):
 
       assert_that(actual, is_buffered_correctly)
 
+  def test_deduplication_transform_with_processing_time(self):
```

Comment:
fn_api_runner_test primarily for testing primitives. I would put this in apache_beam.transforms.deduplicate_test (or util_test, if that's where we plan to move it).